webots_ros2_core/webots_ros2_core/devices/robot_device.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 1996-2020 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Robot device."""
+
+import os
+import tempfile
+from .device import Device
+from rcl_interfaces.srv import SetParameters
+from rcl_interfaces.msg._parameter import Parameter
+from rclpy.parameter import ParameterType, ParameterValue
+
+
+class RobotDevice(Device):
```

Comment:
Even if inherits from `Device` I would not name the class like this, but rather something that really explains what the class is doing, e.g. `RobotDescriptionPublisher`.

Reply:
The idea is to further extend this class later with new functionalities, e.g. battery status.

Reply:
ok, in that case we can leave it as is but explain clearly what it currently does in the class description.